src/autocomplete/databases/mysql/tests/extract-statement-positions.test.ts

```diff
@@ -0,0 +1,163 @@
+import {StatementPosition} from '../../../shared/extract-statement-positions-from-query';
```

Comment:
we usually write tests for common modules once, but I think it's important enough for us to have tests for all clusters
@NikitaShkaruba what do you think?